src/control/path_follower/path_follower/node_particle_pursuit_inherit.py

```diff
@@ -240,6 +239,35 @@ def publish_debug_image(self, steering_angle: float, velocity: float, rvwp: List
 
         self.debug_publisher.publish(cv_bridge.cv2_to_imgmsg(debug_img, encoding="bgr8"))
 
+    def publish_debug_image(self, steering_angle: float, velocity: float, rvwp: List[float], position: List[float]):
```

Comment:
Just noticed this function is here twice? the only change is the publisher name, which i think is whats breaking

Reply:
That was not the issue...

Reply:
You talking about the issue in the other comment?

Reply:
yeah i was. worked it out tho